packages/web-components/fast-components-msft/src/menu/menu.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+    elevation,
+    neutralDividerRestBehavior,
+    neutralLayerFloatingBehavior,
+} from "../styles/index";
+
+export const MenuStyles = css`
+    ${display("block")} :host {
+        --elevation: 11;
+        background: ${neutralLayerFloatingBehavior.var};
+        border: calc(var(--outline-width) * 1px) solid transparent;
+        border-radius: var(--elevated-corner-radius);
+        ${elevation} margin: 0;
```

Comment:
Is `elevation` correct here?

Reply:
That does looks wrong, I copied this from `fast-components` and made the necessary color changes. so its wrong there to.

Reply:
needs a semicolon and a return